### DISCUSSION TITLE
Is there any Complete installation Tutorial for Windows ?

### DISCUSSION THREAD

**Discussion by asusdk** [Comfy-Org/ComfyUI-Manager#1178] [General]
Hi @ all,

i am a normal Windows User, and have not much experience with python. I used the Portable Version of Comfy, but I get errors over errors with it. 

  I created a Folder in C:\ComfyUI and extracted the files from the Archive. I installed the Manager. And also installed the tensorrt Costum node.

I just can't get it to work. I googled, I used ChatGPT I did nearly everything, but I can't get it to work.

If I rarely find something that is supposed to help, all people talk about some VENV folder, but there is no such folder in the Comfy Directory. After another round of Google, it seems to be some Virtual environment or Something. But the Standalone seems not to use one ?

Is there ANY complete step-by-step tutorial How to set Comfy up, so that I can finally get the Nodes running correctly ?

I watched now around 8 different videos on Youtube with the Search phrase "Install ComfyUI" but all i ever find are videos of people that just download the standalone, and they never seem to have any errors.

I'm a lot frustrated right now... can please someone help me ?

kind regards

edit: 
Installed Python on my Machine: 3.10.11

Error of tensorrt at the startup:
Cannot import C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_TensorRT module for custom nodes: No module named 'tensorrt_bindings'

the solution I found would be:
https://github.com/comfyanonymous/ComfyUI_TensorRT/issues/18

But I can't figure out how to use that with the portable version, so I think that I need some other sort of installation ?

Edit2:
I had also tried:
an older python version: 3.10.6
a Newer python version  (sadly forgot to note which one it was, I think 3.12 or something)
but as the most recommended seems to be the 3.10.11, so I settled for this one
Installed Cuda: 12.6
Graphics Card: RTX4070
Win11

**Comment by ltdrdata** [comment]
Are you looking for ComfyUI or ComfyUI powered by TensorRT?

If you simply want ComfyUI, there's no need to install ComfyUI_TensorRT.

If you want ComfyUI powered by TensorRT, you should inquire at this repo:
https://github.com/comfyanonymous/ComfyUI_TensorRT/issues
